models/cobalt/model.sdf

```diff
@@ -168,6 +173,75 @@
       </velocity_decay>
     </link>
 
+    <link name='hydrophones'>
+      <pose> 0 0 0 0 0 0 </pose>
+      <inertial>
+        <mass> 0.001 </mass>
+      </inertial>
+      <visual name='ref_hydrophone_visual'>
+        <transparency>0.7</transparency>
+        <pose> 0.5 0 0 0 0 0</pose>
```

Comment:
Is there any reason for these particular hydrophone positions or is it just arbitrary until we get them on the real sub?